cmd/operator/main.go

```diff
@@ -42,6 +48,11 @@ func init() {
 }
 
 func main() {
+	err := observability.Start(otelServiceName)
+	if err != nil {
+		log.Printf("Could not start OpenTelemetry: %s", err)
```

Comment:
"Printf" needs a return at the end of the line:

```suggestion
		log.Printf("Could not start OpenTelemetry: %s\n", err)
```

Reply:
Fixed.